download_dataset.py

```diff
@@ -0,0 +1,26 @@
+"""Script to download Sentiment140 dataset."""
+
+import argparse
+import io
+import zipfile
+
+import requests
+
+import utils
+
+DATASET_URL = "http://cs.stanford.edu/people/alecmgo/trainingandtestdata.zip"
+
+parser = argparse.ArgumentParser(
```

Comment:
The parser is only necessary when runing the script directly; therefore I would put it in the `if __name__ == '__main__'` condition

Reply:
Corrected in https://github.com/XanX3601/deep_sentiment140/pull/3/commits/4833fdfb78d9b6d5d7314cc5c237879d8ad4c360.